starlite/openapi/parameters.py

```diff
@@ -26,11 +27,13 @@ def create_parameters(
     """
     path_parameter_names = [path_param["name"] for path_param in path_parameters]
     parameters: List[Parameter] = []
-    ignored_fields = [
-        *RESERVED_KWARGS,
-        *list(route_handler.resolve_dependencies().keys()),
-    ]
+    ignored_fields = [*RESERVED_KWARGS]
```

Comment:
no need to unpack this, or to aliad it as `ignored_fields` if its only this iterable.

Reply:
Ah, good catch, will make a small change to improve this.